src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -2072,18 +2044,24 @@ public function import_aggregator_image( $event ) {
 	}
 
 	/**
-	 * Imports the image contained in the event `image` field if any.
+	 * Imports the image contained in the post data `image` field if any.
 	 *
-	 * @param array $event An event data in array format.
+	 * @param array $data A post data in array format.
 	 *
 	 * @return object|bool An object with the image post ID or `false` on failure.
 	 */
-	public function import_image( $event ) {
-		if ( empty( $event['image'] ) || ! filter_var( $event['image'], FILTER_VALIDATE_URL ) ) {
+	public function import_image( $data ) {
+		if (
+			empty( $data['image'] )
+			&& (
```

Comment:
:thinking: ... if `$data['image']` is empty surely the result of the next part of the conditional will also always be true? Is the second part needed, or else should we be doing something other than `&&` here?

Reply:
You are correct! I fixed this and pushed.